astro/src/content/docs/customize/look-and-feel/upgrade.mdx

```diff
@@ -20,6 +20,16 @@ New versions of FusionAuth sometimes include new or updated theme templates. If
 
 This page contains notes on changes in recent versions of FusionAuth that require changes to a customized theme.
 
+## Version 1.53.3
+
+Version `1.53.3` includes a change to persist the user's SSO decision through an external identity provider workflow. If the Google IdP's `loginMethod` is configured as `UsePopup` or `UseVendorJavaScript`, existing custom advanced themes require an update to the `helpers` template to incorporate the fix for the Google IdP. You can update the template via the API or by modifying the *Helpers* template in the admin UI.
```

Comment:
That is correct. We just want to be careful to not use names for things that aren't documented, or could be understood in more than one way. 

So in this case, I think you are referring to `theme.templates.helpers` on the API or `Helpers` in the UI. What I am trying to avoid is having to have the same thing described in various ways to avoid confusion.

If you wanted to reference `helpers` as you have it, then I think we would want to provide context by saying, the `helpers` field on the `theme.templates` object, or something like that - perhaps more polished than my example. 

We want to avoid ambiguity if we can. If I put myself in the shoes of an engineer reading this doc, if I don't know a lot about the Theme API, I don't want to have to guess, so if we can be specific about which API field we are referring to, or the field in the admin UI by the exact name, this is useful for the end user.